roles/StackStorm.st2web/defaults/main.yml

```diff
@@ -10,3 +10,6 @@ st2web_ssl_certificate_key: null
 
 # String with a custom nginx configuration file to replace st2.conf. If not provided, the default st2.conf will be used.
 st2web_nginx_config: null
+
+# StackStorm version to install. `present` to install available package, `latest` to get automatic updates or pin it to numeric version like `2.2.0`.
+st2_version: 'latest'
```

Comment:
If we want to split `st2web` to be independent from `st2` role, another var `st2web_version` needs to be added, instead of relying on `st2_version` from `st2` role.

Based on that, `st2_version` needs to be removed from the plays. Check `st2chatops` role for the hint, which follows the same approach.

Reply:
Good catch, fixing now!